epaint/src/texture_atlas.rs

```diff
@@ -17,19 +17,10 @@ impl Texture {
     }
 
     /// Returns the textures as `sRGBA` premultiplied pixels, row by row, top to bottom.
-    ///
-    /// `gamma` should normally be set to 1.0.
-    /// If you are having problems with egui text looking skinny and pixelated, try
-    /// setting a lower gamma, e.g. `0.5`.
-    pub fn srgba_pixels(&'_ self, gamma: f32) -> impl Iterator<Item = super::Color32> + '_ {
+    pub fn srgba_pixels(&'_ self) -> impl Iterator<Item = super::Color32> + '_ {
```

Comment:
As noted elsewhere, I think we should revert the changes to the WebGL1 pipeline, so this code block also needs to be reverted.